packages/react-vanilla-components/src/components/FileUpload.tsx

```diff
@@ -87,38 +121,61 @@ const FileUpload = (props: PROPS) => {
         errorMessage={props.errorMessage}
       >
         <br />
-        <label htmlFor={id} className="cmp-adaptiveform-fileinput__widgetlabel">Attach</label>
-        <input
-          ref={fileInputField}
-          className="cmp-adaptiveform-fileinput__widget"
-          id={id}
-          type='file'
-          name={name}
-          onChange={handleNewFileUpload}
-          required={required}
-          accept={accept?.toString()}
-          multiple
-          max-file-size={maxFileSize}
-          style={{ display: 'none' }}
-        />
-        <ul className="cmp-adaptiveform-fileinput__filelist">
-          {files && files?.map((item: FileObject, index) => (
-            <li className="cmp-adaptiveform-fileinput__fileitem" key={item?.name}>
-              <span className="cmp-adaptiveform-fileinput__filename">{item?.name}</span>
-              <span
-                onClick={() => removeFile(index)}
-                className="cmp-adaptiveform-fileinput__filedelete"
-                aria-label={item?.name}
-                role="button"
-              >
-                x
-              </span>
-            </li>
-          ))}
-        </ul>
+        <div className="cmp-adaptiveform-fileinput__container">
+          <div
+            className="cmp-adaptiveform-fileinput__dragarea"
+            onDragOver={handleDragOver}
+            onDrop={handleNewFileUpload}
+          >
+            <div className="cmp-adaptiveform-fileinput__icon"></div>
+            <div className="cmp-adaptiveform-fileinput__dragtext">
+              {properties?.dragDropText}
+            </div>
+            <label
+              htmlFor={id}
+              className="cmp-adaptiveform-fileinput__widgetlabel"
+            >
+              Attach
```

Comment:
Why this 'Attach' text is hard coded it must be bind to json value?

Reply:
The button text is not received in json